packages/blockchain/base/src/lib/__tests__/base.nft.spec.ts

```diff
@@ -29,7 +29,7 @@ describe.skip('BaseSDK - nft', () => {
     deployNFTSmartContract: [
       api.nftDeployErc721,
       {
-        chain: Currency.BASE,
+        chain: 'BASE',
```

Comment:
Why not change to Currency.ETH_BASE?

Reply:
@Smrecz because nftDeployErc721 requires to pass chain. but in unit tests currency is passed.
previouly currency.base and chain.base had the same value: 'BASE' but now currency.eth_base has value 'eth_base', so it does not matches chain anymore.

I do not understand why in unit tests currency is passed when parameter is chain.